config/locales/en/unwithdraw/unwithdraw.yml

```diff
@@ -1,4 +1,7 @@
 en:
   unwithdraw:
+    no_managing_editor_permission:
+      title: Only a managing editor can unwithdraw a document
+      content: If you think this content should be unwithdrawn on GOV.UK, speak to the managing editor for your organisation.
```

Comment:
Similar vein as above comment:

"If you think this content should have the withdrawn banner removed, speak to the managing editor for your organisation."